srcpkgs/olm/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'olm'
 pkgname=olm
-version=2.3.0
+version=3.0.0
 revision=1
 build_style=gnu-makefile
 make_check_target=test
-hostmakedepends="python-devel python3-devel"
-makedepends="python-devel python3-devel"
-short_desc="An implementation of the Double Ratchet cryptographic ratchet"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel libffi-devel"
+depends="python-cffi python3-cffi python-future python3-future"
+short_desc="Implementation of the Double Ratchet cryptographic ratchet"
 maintainer="Adam Beckmeyer <adam_gpg@thebeckmeyers.xyz>"
 license="Apache-2.0"
 homepage="https://git.matrix.org/git/olm/about/"
 distfiles="https://git.matrix.org/git/olm/snapshot/${pkgname}-${version}.tar.gz"
-checksum=533714fb84860e04c185790d16ef9085f15e902c2105db941d5c7e92b0565ef8
+checksum=4c54f93e4f426a9d5c8e66e63113d42fd2afca240cdcff30a469a342a6ec3bc6
+nocross=yes
```

Comment:
why mark it nocross? It was possible to cross-compile before, the package is in the aarch64 repo.

Reply:
During the install stage it uses the host gcc instead of target’s

Reply:
An it isn't possible to find a way around that or patch it to do the right thing? Adding nocross to new packages that simply cannot be crosscompiled is one thing but setting existing packages nocross means the repositories have to be cleaned up manually and people already using a package on cross-compiled architectures will loose functionality.

Reply:
I will try more, as I was trying to avoid setting nocross